Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -305,6 +305,17 @@ public virtual GoogleCredential CreateScoped(IEnumerable<string> scopes)
         public virtual GoogleCredential CreateWithQuotaProject(string quotaProject) =>
             new GoogleCredential(credential.WithQuotaProject(quotaProject));
 
+        /// <summary>
```

Comment:
So if the existing credential has a quota project in, then this can change it to a new value, but can never "remove" the quota project. That's probably fine, but it might be worth highlighting in remarks - again, doesn't have to be right now.

Reply:
Yes, I've confirmed this behaviour with the Auth team. I'm adding remarks.